logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/incremental/EventGatherer.kt

```diff
@@ -152,7 +152,9 @@ internal class EventGathererImpl(
 
     private suspend fun FlowCollector<EventEnvelope>.onWebSocketOpen() {
         logger.i("Websocket Open")
-        handleTimeDrift()
+        // TODO: Handle time drift in a different way, e.g. the notification api is already called
+        //  somewhere else so maybe we can take the time from there ?
+//          handleTimeDrift()
```

Comment:
This is a great point! We need to find another way, since for async notifications this won't work too as it is relying on `/notifications` endpoints, which is not available for consumable-notifications capable clients (api >=  v8 eventually)